### PR TITLE
updated indents

### PR DESCRIPTION
**Pull request by david-gibbs-ig**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -59,3 +59,5 @@ target/
 
 # Mac OS
 .DS_Store
+
+**.bak
```
Comment (femtotrader) [inline .gitignore]: why **?

Comment (david-gibbs-ig) [reply .gitignore]: Sorry femtotrader, could you ignore this pull request please. It is not meant to be public yet, experimental feature. I will share as soon as possible.

**Comment by david-gibbs-ig** [comment]
Should not be submitted to this public project yet.

File: .gitignore (hunk 1 above)
Comment (femtotrader) [reply .gitignore]: No problem

**Pull request closed without merging**